src/main/drivers/resource.h

```diff
@@ -120,4 +122,5 @@ typedef struct resourceOwner_s {
 extern const char * const ownerNames[OWNER_TOTAL_COUNT];
 
 #define RESOURCE_INDEX(x) (x + 1)
+// TODO(hertz@): only used by vtx_rtc6707_soft_spi and probably for display purposes
 #define RESOURCE_SOFT_OFFSET    10
```

Comment:
This constant is now used in one place only for display purposes, for displaying OWNER of softspi vtx.
I think it can safely be removed or moved to the site of usage.

Reply:
Agree. Probably move `RESOURCE_SOFT_OFFSET` define to `drivers/vtx_rtc6705_soft_spi.c`